apps/betterangels-backend/clients/tests/test_permissions.py

```diff
@@ -282,6 +285,6 @@ def test_view_client_documents_permission(self, user_label: str, should_succeed:
             )
         else:
             self.assertTrue(
-                len(response["data"]["clientDocuments"]) == 0,
+                len(response["data"]["clientDocuments"]["results"]) == 0,
```

Comment:
actually, with the above comment in mind, if you wanted to, you could replace `len()` with `totalCount`
```suggestion
               response["data"]["clientDocuments"]["totalCount"] == 0,
```

Reply:
come to think of it, I'm not sure why this was originally written with a `assertTrue`. Should have been `self.assertEqual(<len_results>, 0)`